guides/common/attributes-base.adoc

```diff
@@ -187,7 +187,7 @@
 :client-content-content-view-label: AlmaLinux_9
 :client-content-product-label: {client-content-content-view-label}
 :client-content-repository-label: BaseOS
-:client-salt-minion-repository-url: https://repo.saltproject.io/py3/
+:client-salt-minion-repository-url: https://packages.broadcom.com/ui/repos/tree/General/saltproject-rpm
```

Comment:
My patch was wrong, sorry for that. I believe that the attribute is indeed unused, you were right the first time.

Reply:
It was used in `guides/common/modules/proc_providing-the-salt-client-to-salt-minions.adoc` but your patch deleted it. So I dropped it from here too.